### PR TITLE
OSDOCS-5232-4-12: Updated the 4.12 RNs to state vSphere 8.0 support

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-5232](https://issues.redhat.com/browse/OSDOCS-5232)

Version(s):
4.12

Link to docs preview:
[Preview link](http://file.emea.redhat.com/dfitzmau/OSDOCS-5232-4-12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-post-installation)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -3608,8 +3608,6 @@ $ oc adm release info 4.12.15 --pullspecs
 [id="ocp-4-12-15-updating"]
 ==== Updating
 
-<<<<<<< HEAD
-=======
```
Comment (dfitzmau) [inline release_notes/ocp-4-12-release-notes.adoc:3612]: Including a fix for https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-15

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**